app/models/effective/event_addon.rb

```diff
@@ -42,6 +42,8 @@ class EventAddon < ActiveRecord::Base
 
     scope :sorted, -> { order(:id) }
     scope :deep, -> { includes(:event, :event_product) }
+    scope :archived, -> { where(archived: true) }
+    scope :unarchived, -> { where(archived: false) }
 
```

Comment:
the `acts_as_archived` concern above already adds `.archived` and `.not_archived`  I think we could remove these please